src/components/common/AddNameTranslation.vue

```diff
@@ -1,40 +1,38 @@
 <template>
-  <div id="name-translation">
+  <div id="add-name-translation">
     <!-- Name Translation form -->
-    <v-form v-model="nameTranslationForm" class="name-translation-form">
-      <v-row>
-        <v-col class="pb-0">
-          <v-text-field
-            filled
-            persistent-hint
-            id="name-translation-input"
-            label="Enter Name Translation"
-            v-model="nameTranslation"
-            :rules="nameTranslationRules"
-          />
-        </v-col>
-      </v-row>
-      <v-row>
-        <v-col class="form__btns pt-0">
```

Comment:
It's silly to use rows and columns for this when `form__btns` already implements flex, which gives us the formatting we need.